pkg/proto/pbgo/core/api.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.31.0
-// 	protoc        v3.21.5
+// 	protoc-gen-go v1.28.1
```

Comment:
It's probably safest to exclude these unrelated files from your change list.

Reply:
@truthbk explained that this should be fine - will tag him for review.